### ISSUE TITLE
/forcesellall feature

### ISSUE THREAD

**Issue by freshfunkee** [gcarq/freqtrade#32]
Hey guys,

This is a really neat project. Thanks for all your work. I've been tinkering around with the code, although I have literally no knowledge of python previous to writing some small changes to this repo.

I parsed some of the logic together and created my own fork for a `/forcesellall` feature. This is something that I thought would be useful to get everything back to bitcoin (for example, when you reach a certain ROI of profit)

This could be expanded further through the config as a "sell all" once a certain ROI has been achieved, or when a certain number of trades have completed.

For now, this is just the manual telegram command. Here it is on a feature branch on my fork:
https://github.com/freshfunkee/freqtrade/blob/feature/forcesellall/rpc/telegram.py

I'm posting it here as I would love some input into creating a test for this function in the existing `test_telegram.py` before submitting a pull request. As I've mentioned, I have little to no knowledge of python. I'll be poking around in the meantime, but if someone could explain one or two of the existing tests and what's going on there, I would be able to give this a stab.

**Comment by gcarq** [comment]
Hi,

I don't know if you're familiar with data mocking, but if you look into `test_3_forcesell_handle`, you can see that there are multiple mocks in place (`patch.dict(...), ...`), these are necessary to patch some functions during runtime to not rely on external APIs.

You can read more about the mock library [here](https://docs.python.org/3/library/unittest.mock.html)

if you take a look at
```
with patch.multiple('main.telegram', _CONF=self.conf, init=MagicMock(), send_msg=msg_mock):
    ...
```
What it does is searching for the module `telegram` within `main` and replaces the variables `_CONF`, `init` and `send_msg` there with some local variables.

The remaining code is just creating a trade, executing `/forcesell 1` and assertions that the expected state has achieved.
You can pretty much clone this test, create more different trades, execute `/forcesellall` and assert

**Comment by gcarq** [comment]
@freshfunkee I cherry picked your commit and refactored it into `/forcesell`.
The syntax now is `/forcesell (trade_id|all)`. It has been released with 0.14.1
